app/models/person.rb

```diff
@@ -103,6 +103,8 @@ class Person < ActiveRecord::Base
   has_many :authored_comments, :class_name => "Comment", :foreign_key => "author_id", :dependent => :destroy
   has_many :community_memberships, :dependent => :destroy
   has_many :communities, -> { where("community_memberships.status = 'accepted'") }, :through => :community_memberships
+  has_one  :community_membership, :dependent => :destroy
+  has_one  :community, -> { where("community_memberships.status= 'accepted'") }, through: :community_membership
```

Comment:
What if we go through a middle ground here, where wouldn't change the logic, but rename the scope `community` to `accepted_community`? That way we would only have to rename `communities` to `accepted_community` and remove the iteration of the collections. 

I think that this would better tell what's the purpose of the member.